### PR TITLE
add skip-tls-verify flag for insecure private registries

### PR DESCRIPTION
**Pull request by beniamin**
Added a new flag to skip https certificate verification for private docker registries.

### CODE REVIEW

File: cmd/kaniko-docker/main.go [hunk 1]
```diff
@@ -94,6 +94,12 @@ func main() {
 			Usage:  "docker password",
 			EnvVar: "PLUGIN_PASSWORD",
 		},
+		cli.StringFlag {
```
Comment (bradrydzewski) [inline cmd/kaniko-docker/main.go:97]: you can use `cli.BoolFlag` instead

File: cmd/kaniko-docker/main.go [hunk 2]
```diff
@@ -94,6 +94,12 @@ func main() {
...
+			Name: "skip_tls_verify",
```
Comment (bradrydzewski) [inline cmd/kaniko-docker/main.go:98]: use kebab case for flag names (skip-tls-verify)

File: cmd/kaniko-docker/main.go [hunk 3]
```diff
@@ -116,6 +122,7 @@ func run(c *cli.Context) error {
 			Target:     c.String("target"),
 			Repo:       c.String("repo"),
 			Labels:     c.StringSlice("custom-labels"),
+			SkipTlsVerify: c.String("skip_tls_verify"),
```
Comment (bradrydzewski) [inline cmd/kaniko-docker/main.go:125]: once you chance to `cli.BoolT` you can use `c.Bool("skip-tls-verify")` which returns a bool value

File: kaniko.go [hunk 4]
```diff
@@ -17,6 +17,7 @@ type (
 		Target     string   // Docker build target
 		Repo       string   // Docker build repository
 		Labels     []string // Label map
+		SkipTlsVerify	string // Docker skip tls certificate verify for registry
```
Comment (bradrydzewski) [inline kaniko.go:20]: change to bool and also run `gofmt` on the codebase, which will automatically format the code.  If you are using an IDE (like vscode) the official Go extension should also autoformat on-save.

**Comment by bradrydzewski** [comment]
👋  hey there, thanks for taking the time to send a pull request! I just added a few minor comments, and once addressed, we would be happy to merge.

File: cmd/kaniko-docker/main.go (hunk 1 above)
Comment (beniamin) [reply cmd/kaniko-docker/main.go]: done

File: cmd/kaniko-docker/main.go (hunk 2 above)
Comment (beniamin) [reply cmd/kaniko-docker/main.go]: done

File: cmd/kaniko-docker/main.go (hunk 3 above)
Comment (beniamin) [reply cmd/kaniko-docker/main.go]: done

File: kaniko.go (hunk 4 above)
Comment (beniamin) [reply kaniko.go]: done

**Comment by beniamin** [comment]
Glad to help.
I solved all the issues that you suggested. 
Please let me know if there is something else I can do.

**Pull request merged**